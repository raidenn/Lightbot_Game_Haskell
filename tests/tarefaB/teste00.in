aaaaaAbbbbbBcccccZ
aaaaaAbbbbbBcccccC
aaaaaAbbbbbBcccccC
aaaaaAbbbbbBcccccC
aaaaaAbbbbbBcccccC
aaaaaAbbbbbBcccccC
aaaaaAbbbbbBcccccC
aaaaaAbbbbbBcccccC
aaaaaAbbbbbBcccccC
aaaaaAbbbbbBcccccC
aaaaaAbbbbbBcccccC
aaaaaAbbbbbBcccccC
aaaaaAbbbbbBcccccC
aaaaaAbbbbbBcccccC
aaaaaAbbbbbBcccccC
aaaaaAbbbbbBcccccC
aaaaaAbbbbbBcccccC
aaaaaAbbbbbBcccccC
17 17 S
SALFNDSFNLSKDFMSMFLSK
