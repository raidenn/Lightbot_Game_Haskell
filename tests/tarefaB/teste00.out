17 16 S
